gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/source/AuthSchemeException.java

```diff
@@ -9,8 +9,18 @@
  */
 package org.zowe.apiml.gateway.security.service.schema.source;
 
-public class UserNotMappedException extends RuntimeException {
-    public UserNotMappedException(String message) {
+public class AuthSchemeException extends RuntimeException {
```

Comment:
Do you think it would make sense to use checked exception here?

Reply:
This exception could be checked, however in some cases AuthSchemeException just wraps authentication exceptions, which are runtime exceptions now. I would make suggested change is separate branch, so that proposed solution can be evaluated and merged later.